contents/docs/experiments/best-practices.mdx

```diff
@@ -0,0 +1,118 @@
+---
+title: Experiment best practices
```

Comment:
Can keep the sidebar the same but this has 10x better SEO traffic.
```suggestion
title: A/B testing best practices
```

Reply:
100% this

Reply:
Sorry but disagree 🙃 We renamed it everywhere from "A/B tests" to "experiments" in the product and docs, so we should continue to refer to it as experiments here in the docs.

In any case, Im dropping the product name from the title and just calling it "Best practices" to keep consistent with other products - see [this comment](https://github.com/PostHog/posthog.com/pull/9858#discussion_r1840252050) on previous PR.

Reply:
+1 on not dropping the name